plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityArms.java

```diff
@@ -3,80 +3,40 @@
 import com.denizenscript.denizen.objects.EntityTag;
 import com.denizenscript.denizencore.objects.core.ElementTag;
 import com.denizenscript.denizencore.objects.Mechanism;
-import com.denizenscript.denizencore.objects.ObjectTag;
-import com.denizenscript.denizencore.objects.properties.Property;
-import com.denizenscript.denizencore.objects.properties.PropertyParser;
 import org.bukkit.entity.ArmorStand;
 
-public class EntityArms implements Property {
+public class EntityArms extends EntityProperty<ElementTag> {
 
-    public static boolean describes(ObjectTag entity) {
-        return entity instanceof EntityTag
-                && ((EntityTag) entity).getBukkitEntity() instanceof ArmorStand;
-    }
+    // <--[property]
+    // @object EntityTag
+    // @name arms
+    // @input ElementTag(Boolean)
+    // @description
+    // Controls whether an armor stand has arms.
+    // -->
 
-    public static EntityArms getFrom(ObjectTag entity) {
-        if (!describes(entity)) {
-            return null;
-        }
-        else {
-            return new EntityArms((EntityTag) entity);
-        }
+    public static boolean describes(EntityTag entity) {
+        return entity.getBukkitEntity() instanceof ArmorStand;
     }
 
-    public static final String[] handledMechs = new String[] {
-            "arms"
-    };
-
-    public EntityArms(EntityTag entity) {
-        dentity = entity;
+    @Override
+    public ElementTag getPropertyValue() {
+        return new ElementTag(as(ArmorStand.class).hasArms());
     }
 
-    EntityTag dentity;
-
     @Override
-    public String getPropertyString() {
-        return getStand().hasArms() ? "true" : null;
+    public void setPropertyValue(ElementTag param, Mechanism mechanism) {
+        if (mechanism.requireBoolean()) {
+            as(ArmorStand.class).setArms(param.asBoolean());
+        }
     }

```

Comment:
The old `getPropertyString` returned `null` for false, as that was the default value (I.e. didn't need to be specified in the entity's property map) - `getPropertyValue` should still return the full boolean so that the `.arms` tag works as expected, but should implement `isDefaultValue` here.